documentation/src/docs/reader-group-design.md

```diff
@@ -11,83 +11,85 @@ You may obtain a copy of the License at
 
 _To be updated to reflect recent implementation details (05/07/2017)_
```

Comment:
@shrids  Any further updation required?

Reply:
yes, please find the summary below
- Reader creation:
   The document talks about a reader creation using the snippet `a_stream.createReader(...)` this has to be replaced with the snippet
```
ClientFactory clientFactory = ClientFactory.withScope(scope, controllerURI);
EventStreamReader<Integer> reader = clientFactory.createReader(readerId, READER_GROUP_NAME, serializer, readerConfig);
```
(for reference use: https://git.io/fxStL)

- Reader going offline (section)
Here the documentation talks about how the last known position of the reader is passed to the `readerOffline(readerId, position) api`.
If the position is sent as "null" then the last checkpointed position of the reader is used by the newer readers when they take ownership of the segment(s) that were read by the older/offline reader.

Reply:
can i mention position object or parameter